### DISCUSSION TITLE
HM-600 reports power factor > 1

### DISCUSSION THREAD

**Discussion by avandorp** [tbnobody/OpenDTU#1371] [Q&A]
Hi all. OpenDTU sometimes reports a power factor slightly bigger than 1, e.g 1.002, or even 1.019, defying the laws of physics. :) Is this just some strange calculation error coming from the inverter or is there more to it? It is usually at 1.000. Reactive power is ususally 0 VAr, sometimes up to 0.3 VAr.

**Comment by avandorp** [comment]
It looks as if values >1 usually occur during low light situations where the inverter almost shuts off. My working theory is, that it is merely bad floating point arithmetic. That's a hard problem even on a "big" computer but difficult on these embedded systems.
Power factorin the late evening:
![image](https://github.com/tbnobody/OpenDTU/assets/18458913/eec83900-8f8b-4c0f-a763-7cd7624fb922)

**Comment by CBFOUR** [comment]
Jep, the same on my HM-600, when he is low on DC, short bevor he "Die" , i also see funny values.
But iam pretty shure, it is the Inverter who send this values. 
![powerfactor](https://github.com/tbnobody/OpenDTU/assets/128149877/cab334d8-b673-4f8d-8439-b5fcddf978e1)
